skyportal/handlers/api/photometry_validation.py

```diff
@@ -0,0 +1,323 @@
+from marshmallow import Schema, fields, validates_schema
+from marshmallow.exceptions import ValidationError
+
+from baselayer.app.access import permissions
+from ..base import BaseHandler
+
+from ...models import Photometry, PhotometryValidation
+
+
+class Validator(Schema):
+    method = fields.Str(required=True)
+    photometry_id = fields.Integer()
+    validated = fields.Boolean(
+        truthy=['true', 'True', 'validated', True],
+        falsy=['false', 'False', 'rejected', False],
+        required=False,
+    )
+    explanation = fields.String(required=False)
+    notes = fields.String(required=False)
+
+    @validates_schema
+    def validate_requires(self, data, **kwargs):
+        if 'method' not in data:
+            raise ValidationError('method is required')
+        if data['method'] not in ['POST', 'PATCH', 'DELETE']:
+            raise ValidationError('method must be one of POST, PATCH or DELETE')
+        if (
+            data['method'] == 'PATCH'
```

Comment:
`if data['method'] in ('PATCH', 'DELETE', 'POST')`

Reply:
fixed.